open-media-match/src/OpenMediaMatch/templates/index.html.j2

```diff
@@ -43,7 +43,7 @@
         <!-- Add Content to Bank -->
         <h3> Add Content to Bank: </h3>
         <form action = "/ui/addcontent" method = "post" enctype="multipart/form-data">  
-            <input type="file" name="file" /> Banks
+            <input type="file" name="photo" /> Banks
```

Comment:
We should switch this over the a selector.

You might be able to rewrite the request parameters to populate it how the hash expects.